src/Authentication/Authentication/Helpers/WebResponseHelper.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net.Http;
+using Microsoft.Graph.PowerShell.Authentication.Cmdlets;
+
+namespace Microsoft.Graph.PowerShell.Authentication.Helpers
+{
+    internal class WebResponseHelper
+    {
+        internal static string GetCharacterSet(HttpResponseMessage response)
+        {
+            var characterSet = response.Content.Headers.ContentType.CharSet;
+            return characterSet;
+        }
```

Comment:
consider making this an extension method by changing the argument to `(this HttpResponseMessage response)`